### PR TITLE
VS from CAG: changes in response to review comments

### PR DESCRIPTION
**Pull request by lantunes**
These changes are in response to a previous review, that was missed.

Previous comments:
> What is the purpose of this variable? Maybe a more meaningful name than qu would help the reader to better understand.

I have updated the code so that it is, hopefully, more readable.

> In models like SIR, we don't have this key so I think we need to check if metadata key exists first.
> `if ('metadata' in model):`

I am now throwing an exception if the top-level `metadata` field is missing from the CAG, since we cannot do the analysis without it.

> I think we might want to give an example of how a minio hostname would look like (without revealing it). I've struggled to figure out the right hostname for a bit.

I have added an example to the `README` file.

### CODE REVIEW

File: EPI/create-VS-from-CAG.py [hunk 1]
```diff
@@ -41,9 +45,13 @@ def analyze_cag(input):
     with open(input, "rt") as f:
         model = json.load(f)
 
+        if "metadata" not in model:
```
Comment (RosaRomeroGomez) [inline EPI/create-VS-from-CAG.py:48]: So we can't generate the SIR CAG without metadata then?

Comment (lantunes) [reply EPI/create-VS-from-CAG.py:48]: We're using the top-level metadata to identify the variables and constants properly. I suppose we can use the fact that a node has no incoming edges, but I'd have to have a closer look to see if that works for the simple SIR and double epi models.

Comment (RosaRomeroGomez) [reply EPI/create-VS-from-CAG.py:48]: Let's merge this one and we can address that in a subsequent PR. I'd like to incorporate the 3 VS to the HMI to see how that looks in the near future.

Comment (lantunes) [reply EPI/create-VS-from-CAG.py:48]: Ok, this has been merged. But just to be clear, incorporating the 3 VS we have doesn't depend on this PR correct? We can continue to incorporate the 3 VS we had previously created manually.

**Pull request merged**